msteams-platform/toolkit/TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -134,13 +134,13 @@ This option renders `appPackage/manifest.json` with environment variables, and t
 
 :::image type="content" source="../assets/images/teams-toolkit-v2/customize app manifest/validate-schema.png" alt-text="Screenshot shows the selection of validate using manifest schema.":::
 
-Alternatively, use the following Microsoft Teams Toolkit command line interface (Teams Toolkit CLI) command:
+Alternatively, use the following Agents Toolkit command line interface (Agents Toolkit CLI) command:
 
 ```bash
 teamsfx validate --manifest-path <YOUR-PATH-TO-MANIFEST>
```

Comment:
cli has been changed to "atk", seems the whole file needs to be revisited to check whether the bash sample use the correct command name.